addon/webextension/background/selectorLoader.js

```diff
@@ -80,21 +85,43 @@ this.selectorLoader = (function() {
     });
   };
 
+  function readFile(path) {
+    return fetch(path, { mode: 'same-origin' }).then((response) => {
+      return response.blob();
```

Comment:
Can this be `response.text()` instead of `response.blob()`, skipping the FileReader stuff?